config/codon_dbs.config

```diff
@@ -1,5 +1,5 @@
 params {
-    gtdb_db = "/hps/nobackup/rdf/metagenomics/service-team/production/ref-dbs/genomes-pipeline/release220"
+    gtdb_db = "/hps/nobackup/rdf/metagenomics/service-team/production/ref-dbs/genomes-pipeline/release226"
```

Comment:
you should move codon configs to private configs repo, I'm happy to help with that